static/bidder-params/kargo.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Kargo Adapter Params",
+    "description": "A schema which validates params accepted by the Kargo adapter",
+    "type": "object",
+  
+    "properties": {
+        "placementId": {
+          "type": "string",
+          "description": "Placement ID"
+        }
+      },
+    
+    "required": ["placementId"]
```

Comment:
Neither `adapters/kargo/kargotest/exemplary/banner.json` nor `adapters/kargo/kargotest/exemplary/native.json` seem to come with a `"placementID"` parameter in any of the `request.imp[i].ext` fields. Are we sure this parameter is required?

Reply:
Sorry, this should have been `tagid` instead